OracleTuxedo/performance_pack/dockerfiles/perfpack_runme.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/sh
+#
+# Shell script to build and run perfpack.  It assume TUXDIR has been set, or that 
+# Tuxedo has been installed to: ~/tuxHome/tuxedo12.1.3.0.0   If not, invoke 
+# this script with a single argument indicating the location of TUXDIR.
+#
+# Author: Judy Liu
+#
+# Usage: source perfpack_runme.sh
+#
+cd /u01/oracle/user_projects
+rm -rf perfpack
+mkdir perfpack
+cd perfpack
+
+# Create environment setup script setenv.sh
+export HOSTNAME=`uname -n`
+export APPDIR=`pwd`
+
+cat <<EOF
```

Comment:
You're outputting this at runtime, but really it should be part of the documentation so that it's set properly in the `docker run` command that actually starts the container.